_includes/header/header-th.html

```diff
@@ -113,7 +113,10 @@
                             <a href="/{{ page.lang }}/resources/contributing.html">มีส่วนร่วมกับ Express</a>
                           </li>
                           <li>
-                            <a href="/{{ page.lang }}/changelog/4x.html">บันทึกการเปลี่ยนแปลง</a>
+                            <a href="/en/resources/contributing-docs.html">Contributing to Docs</a>
+                          </li>
+                          <li>
+                            <a href="/en/changelog/4x.html">บันทึกการเปลี่ยนแปลง</a>
```

Comment:
This page was broken in Thai. Added link to en page
![th_changelog](https://github.com/user-attachments/assets/ec8fc9b8-fa13-41f6-9a6b-ef001ee097f8)